docs/cmake-presets.md

```diff
@@ -0,0 +1,272 @@
+
+Select a Test Preset to make it the active Test Preset. This is the `testPreset` that will be used when CTest is invoked to run tests. The active Test Preset is displayed in the status bar.
+
+All Test Presets are required to specify an associated `configurePreset`. CMake Tools will hide Test Presets that don't apply to the active Configure Preset. For more information, see [Test Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#test-preset).
+
+## CMake: run tests
+
+To show or hide individual status bar icons, modify `cmake.statusbar.advanced` in `settings.json`.
+
+## Add new presets
+
+All commands and preset templates modify `CMakePresets.json`. You can add new user-level presets by directly editing `CMakeUserPresets.json`
+Use forward slashes (`/`) for paths in `CMakePresets.json` and `CMakeUserPresets.json`.
+
+## Add new Configure Presets
+
+To add a new Configure Preset to `CMakePresets.json`, run the **CMake: Add Configure Preset** command. This lists several Configure Preset templates, and an option to **[Scan for Compilers]** in the command palette. **[Scan for Compilers]** returns all of the GCC and Clang compilers on your `PATH`, all compilers found in `cmake.mingwSearchDir` and `cmake.emscriptenSearchDirs`, and the latest instances of `cl.exe` installed with Visual Studio.
+
+![Add a configure preset](images/add-configure-preset-vscode.png)
+
+* Select **Inherit from Configure Preset** to inherit from an existing Configure Preset. For more information about inheritance, see [Configure Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#configure-preset).
+* Select the **Toolchain File** template to configure your CMake project with a CMake toolchain file.
+* Select the **Custom** template to configure an empty Configure Preset.
+* Select **[Scan for Compilers]** to search for C/C++ compilers on your machine.
+
+The selected template will be added to `CMakePresets.json` if `CMakePresets.json` exists. Otherwise, the template will be copied into a new `CMakePresets.json`. See [Edit presets](#edit-presets) for more information on editing Configure Presets.
+
+> [!Note] Windows developers: CMake Tools selects the most recent version of `cl.exe` installed by default. You can specify a specific compiler version with the `toolset` option in `CMakePresets.json`. See [Configure Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#configure-preset) and [Toolset Selection](https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2016%202019.html#toolset-selection) for more information. If CMake and your generator are not installed with Visual Studio, then they will need to be on your `PATH` or in the default installation folder.
```

Comment:
Remote last sentence here ("If CMake and your generator... default installation folder."

Reply:
done